index.html

```diff
@@ -1,14 +1,164 @@
+          <button class="button2" type="button"><a target="_blank" href="continue-reading.html">CONTINUE READING</a></button>
+          <br>
+          <br>
+          <br>
+          <br>
+          <br>
+        </div>
+      
+        <div class="main-content1-sidebar">
+          <br>
+          <div class="date-container">
+            <div class="date">JULY 30</div>
+            <h5 class="sidebar-post-title">JUST ANOTHER POST</h5>
+            <p class="sidebar-post">Lorem ipsum dolor sit amet consectetur adipisicing elit. Officiis atque magnam minima, corporis quibusdam et doloribus ipsum ad cum expedita!</p>
+            <br>
+          </div>
+          <br>
+          <br>
+          <div class="date-container">
+            <div class="date">JULY 28</div>
+            <h5 class="sidebar-post-title">AND ANOTHER POST</h5>
+            <p class="sidebar-post">Lorem ipsum dolor sit amet consectetur adipisicing elit. Officiis atque magnam minima, corporis quibusdam et doloribus ipsum ad cum expedita!</p>
+            <br>
+          </div>
+          <br>
+          <div class="date-container2">
+            <div class="date">JULY 24</div>
+            <h5 class="sidebar-post-title">ONE MORE POST</h5>
+            <p class="sidebar-post">Lorem ipsum dolor sit amet consectetur adipisicing elit. Officiis atque magnam minima, corporis quibusdam et doloribus ipsum ad cum expedita!</p>
+            <br>
+          </div>
+          <div class="date-container">
+            <h5 class="sidebar-post-title">SOMETHING OF NOTE</h5>
+            <div class="sidebar-body">
+             <img class="sidebar-pics" src="https://picsum.photos/300/204"/><p class="sidebar-post">Lorem ipsum dolor sit amet consectetur adipisicing elit. Explicabo vel ad quod atque quos natus saepe exercitationem consequatur odit dolores? Mollitia consequatur reprehenderit earum necessitatibus, magnam beatae. Quae ratione odio suscipit harum, iusto repellendus inventore!</p>
+              <br>
+            </div>
+            <button class="button3" type="button"><a target="_blank" href="learn-more.html">LEARN MORE</a></button>
+            <br>
+          </div>
+
+          <h5 class="sidebar-post-title">SOMETHING OF LESS NOTE</h5>
+          <div class="sidebar-body">
+            <img class="sidebar-pics" src="https://picsum.photos/300/205"/><p class="sidebar-post">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Cumque voluptate incidunt non, ullam dicta, nihil sequi, recusandae impedit voluptates aliquid ad! Modi placeat dignissimos molestiae obcaecati, odit maiores voluptatem ex, quam nobis perspiciatis explicabo eveniet.</p>
+          </div>
+          <br>
+          <button class="button3" type="button"><a target="_blank" href="learn-more.html">LEARN MORE</a></button>

```

Comment:
This is another spot that you should remove target="_blank"

Reply:
Using more descriptive class names would benefit any future work being done on this code as well. You want to make them as clear as your variable names when writing JS